sormas-e2e-tests/src/main/java/org/sormas/e2etests/services/api/SampleApiService.java

```diff
@@ -0,0 +1,44 @@
+/*
+ * SORMAS® - Surveillance Outbreak Response Management & Analysis System
+ * Copyright © 2016-2021 Helmholtz-Zentrum für Infektionsforschung GmbH (HZI)
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program. If not, see <https://www.gnu.org/licenses/>.
+ */
+
+package org.sormas.e2etests.services.api;
+
+import com.google.inject.Inject;
+import java.util.Date;
+import java.util.UUID;
+import org.sormas.e2etests.pojo.api.*;
+
+public class SampleApiService {
+
+  @Inject
+  public SampleApiService() {}
+
+  public Sample buildGeneratedEvent(Case caze) {
```

Comment:
I think here you wanted to have buildGeneratedSample right ?

Reply:
yes it is a typo, thank you